arch/ref-arch.md

```diff
@@ -0,0 +1,85 @@
+            Service Provider                                 Aplication Provider
+     |-------------------------------------------|     |--------------------------|
+~~~~~~~~
+{: #withSP title="Video Traffic with SCONEPRO"}
+
+* (a) The Client opts in to the Communication Service Provider's (CSP) SCONEPRO Service
+* (b) The CSP creates an initial SCONEPRO adaptation properties control block for the Client, and provides initial adaptation properties for the Client
+* The Client takes the adaptation properties it has received from the SCONEPRO Monitor into account when requesting ABR video from the Server
+* The Client requests ABR video content from a Server
+* The Server provides the requested ABR video content to the Client
+* The CSP's SCONEPRO Monitor observes whether the Client's incoming packet stream conforms to the adaptation properties provided to the Client
+   * If the incoming packet stream conforms to those properties, the SCONEPRO Monitor takes no action
+   * If the incoming packet stream does not conform to those properties, the SCONEPRO Monitor takes action as it would for any other packet stream
+* (b) If the path characteristics between the Server and Client change, the CSP's SCONEPRO Monitor might send updated adaptation properties to the Client
+
+Notes:
+
+This example assumes a model where the Client performs self-adaptation based on adaptation parameters provided by the CSP that do not attempt to take advantage of Client-specific capabilities. Including Client-specific capabilities might improve the Client experience in the short term, but minimizing the CSP's knowledge of Client capabilities and relying on the Client to conform to generic SCONEPRO properties will make SCONEPRO less likely to ossify, since Client adaptation behavior can change without requiring corresponding changes at the CSP.
+
+The SCONEPRO Monitor doesn't need to detect video flows from a Client using the SCONEPRO Service, doesn't need to watch every single video flow, and doesn't need to shape every single video flow. If a SCONEPRO Monitor needs to detect non-conforming video flows, but also needs to minimize the overhead required to detect these flows, the SCONEPRO Monitor might watch some sample of flows, or top talkers, or some other subset of flows.
+
+The SCONEPRO Monitor can recognize the situation where a Client using the SCONEPRO Service isn't self-adapting based on the SCONEPRO adaptation properties. What the CSP does in response is outside the scope of SCONEPRO.
+
+SCONEPRO adapation properties will often reflect a business relationship between the Client and the CSP, but this isn't necessary, and might also reflect changing network conditions within CSP's network.
+
+(Is this true?) The CSP and CAP may also have a business relationship that includes traffic parameters, but the chartered goals for SCONEPRO don't target providing adapation guidance for the network path between the CSP and the CAP.
```

Comment:
It is true that the communication happens between the application client and the network, but that does not preclude some business agreements. I'm not sure we should be discussing potential business agreements at all.